### ISSUE TITLE
Recognize Chinese speech

### ISSUE THREAD

**Issue by xieyidi** [SeanNaren/deepspeech.pytorch#501]
Recognize Chinese speech. labels.json contains two thousand characters. I know that the last layer of the model is softmax, which is equivalent to dividing the text into 2000 categories. Is such a large amount reasonable?
